fhservices.cfg.j2

```diff
@@ -121,10 +126,18 @@ define service {
 
 define service {
        service_description Storage::Pod::Disk Usage
-       hosts {{ rhmap_router_dns }}
        check_command check_pod_disk_usage!80!90
        use generic-service
-
+       hostgroup_name core,mbaas
        notes One or more pods have volumes which are running out of disk.  You should clean up the files on this pod, if possible, or scale down/up the pod so that it has a fresh filesystem.
        contact_groups rhmapadmins
 }
+
+define service {
+       service_description Database::MongoDB::Health
+       check_command check_mongodb!mongodb-service-1
```

Comment:
See my comment: [https://github.com/feedhenry/nagios-container/pull/6#issuecomment-237584827].  With only one pod, its named mongodb-1, but with a 3 node mongodb replica set, we have 3 services now, mongodb-1, 2, and 3, each with a selector for a single pod.